Source/DafnyCompilers/CompilerRewriter.dfy

```diff
@@ -348,8 +348,8 @@ module CompilerRewriter {
 
     // TODO: not sure it was worth making this opaque
     predicate {:opaque} GEqCtx(
-      eq_value: (WV,WV) -> bool, ctx: Context, ctx': Context
-    )
+        eq_value: (WV,WV) -> bool, ctx: Context, ctx': Context
+      )
```

Comment:
is this better?

Reply:
I'll add this to my final survey as I like the one on the right better. That way,  arguments don't have the same indentation as the body of the predicate/function, which makes it easy to determine the limit between the two.